File: src/model/Commentaire.ts

export class Commentaire{
  private _email !: string;
  private _idService !: number;
  private _contenu !: string;
  private _dateComment !: Date;


  constructor(email: string, idService: number, contenu: string, dateComment: Date) {
    this._email = email;
    this._idService = idService;
    this._contenu = contenu;
    this._dateComment = dateComment;
  }


  get email(): string {
    return this._email;
  }

  set email(value: string) {
    this._email = value;
  }

  get idService(): number {
    return this._idService;
  }

  set idService(value: number) {
    this._idService = value;
  }

  get contenu(): string {
    return this._contenu;
  }

  set contenu(value: string) {
    this._contenu = value;
  }

  get dateComment(): Date {
    return this._dateComment;
  }

  set dateComment(value: Date) {
    this._dateComment = value;
  }
}

File: lancersBackEnd/model/Commentaire.py
from datetime import datetime

class Commentaire:
    def __init__(self, email: str, idService: int, contenu: str, dateComment: datetime):
        self._email = email
        self._idService = idService
        self._contenu = contenu
        self._dateComment = dateComment

    @property
    def email(self) -> str:
        return self._email

    @email.setter
    def email(self, value: str):
        self._email = value

    @property
    def idService(self) -> int:
        return self._idService

    @idService.setter
    def idService(self, value: int):
        self._idService = value

    @property
    def contenu(self) -> str:
        return self._contenu

    @contenu.setter
    def contenu(self, value: str):
        self._contenu = value

    @property
    def dateComment(self) -> datetime:
        return self._dateComment

    @dateComment.setter
    def dateComment(self, value: datetime):
        self._dateComment = value
